stylefunction.js

```diff
@@ -636,7 +636,7 @@ export default function(olLayer, glStyle, source, resolutions = defaultResolutio
           const haloColor = colorWithOpacity(getValue(layer, 'paint', 'text-halo-color', zoom, f), opacity);
           if (haloColor) {
             textHalo.setColor(haloColor);
-            textHalo.setWidth(getValue(layer, 'paint', 'text-halo-width', zoom, f));
+            textHalo.setWidth(getValue(layer, 'paint', 'text-halo-width', zoom, f) + 0.1 * textSize);
```

Comment:
The stroke is applied around the center of the text outline, so I think
```suggestion
            textHalo.setWidth(getValue(layer, 'paint', 'text-halo-width', zoom, f) + * 2);
```
would bring us closer to what mapbox-gl does.

The [spec](https://docs.mapbox.com/mapbox-gl-js/style-spec/#paint-symbol-text-halo-width) also says that the maximum halo is 1/4 of the text size, which should also be taken into account.

Reply:
would a modulo computation be too much performance wise? That could do the trick, something like :
```suggestion
            textHalo.setWidth((getValue(layer, 'paint', 'text-halo-width', zoom, f) * 2) % (0.25 * textSize));
```